pkg/clients/argocd.go

```diff
@@ -33,6 +35,10 @@ import (
 	"github.com/crossplane-contrib/provider-argocd/apis/v1alpha1"
 )
 
+var (
+	credentialsSourceAzureWorkloadIdentity xpv1.CredentialsSource = "AzureWorkloadIdentity"
```

Comment:
I think it makes sense if we put in `apis`.

Reply:
Done!